### PR TITLE
adds csv data for consortium contributors collected via tripetto

### PR DESCRIPTION
**Pull request by jsheunis**
Requested by @Remi-Gau to add the data collected via tripetto to the jupyter book repo

### CODE REVIEW

**Comment by htwangtw** [comment]
Thank you @jsheunis! Just a heads up: we are expecting one or two people to fill this in and might ask you to update this PR on Monday the latest.

**Comment by SNeuroble** [comment]
Thanks @jsheunis !

Just took a look and I think these will be the next steps updating the authors + affiliations list. Not sure who is interested in doing this but I'm willing to help where I can! Albeit a little busy atm.

1.  extract columns "First and Last Name" (2) and "Affiliation" (4)
2. split column "First and Last Name" into 3 columns: "First", "Middle", and "Last"
3. rename "Affiliation" to "Department" (no need to split unless people feel inspired)
4. manual curation of names and affiliations (mainly: some lack city/state 😕 )
5. merge columns with the existing table
(5.5. To automate co-first stars, we can add an "Other" field + stars to the original table. This is parsed by Author Arranger)
6. sort consortium names alphabetically
7. [Author Arranger](https://authorarranger.nci.nih.gov/#/web-tool)!

**Review by SNeuroble — APPROVED** [review]
By the way, I'm fine with this being merged but we can wait for any updates if need be.

**Comment by htwangtw** [comment]
@SNeuroble I am on this - just waiting for the call :sunglasses:

**Comment by SNeuroble** [comment]
@htwangtw As usual, thank you for being amazing ❤️

**Comment by htwangtw** [comment]
The input for Author Arranger is ready. @jsheunis if there's any new entry please append them to the existing file as I already deleted a few duplicates.

**Comment by jsheunis** [comment]
@htwangtw I've added the new data. hope I didn't screw things up...

**Review by htwangtw — APPROVED** [review]
We are not accepting any more entry for the paper, I hope. 

I am not going to proofread this too hard. We can send `data/authors_affiliations.tsv` to the copy editor when it is time.

File: data/authors_affiliations.tsv [hunk 1]
```diff
@@ -84,137 +84,195 @@ ranking	Aff_Order	First	Middle	Last	Department	Institute	City	Country
...
+95	1	Alexandre	R.	Franco	Center for Biomedical Imaging and Neuromodulation	Nathan Kline Institute	Orangeburg NY	USA
+95	2				Center for the Developing Brain	Child Mind Institute	New York City	USA
+95	3				Department of Psychiatry	NYU Grossman School of Medicine	New York City	USA
+96	1	Saampras	 	Ganesan	Systems Neuroscience	Melbourne Neuropsychiatry Centre	Melbourne	Australia
+96	2				Department of Biomedical Engineering	The University of Melbourne	Melbourne	Australia
+97	1	Siyuan	 	Gao	Yale University	 	 	 
+98	1	José	C.	García Alanis	Department of Psychology	Philipps Universität	Marburg	Germany
+99	1	Eleftherios	 	Garyfallidis	Intelligent Systems Engineering	Indiana University	Bloomington IN	USA
+100	1	Tristan	 	Glatard	Department of Computer Science and Software Engineering  	Concordia University	Montréal, Québec	Canada
+101	1	Enrico	 	Glerean	Department of Neuroscience and Biomedical Engineering, Aalto University, Espoo, Finland; International Laboratory of Social Neurobiology, Institute of Cognitive Neuroscience, National Research University Higher School of Economics, Moscow, Russia
```
Comment (Remi-Gau) [inline data/authors_affiliations.tsv:139]: He should probably be 2 different affiliations

Comment (htwangtw) [reply data/authors_affiliations.tsv]: Cool I can fix that

Comment (htwangtw) [reply data/authors_affiliations.tsv]: So this is the only case of trippetto entry with two affiliations. I decided not to address this in the script as it's just a one off. However, this is reflected on the data used for Jupyter Book page build.

**Comment by htwangtw** [comment]
The Jupyter Book data is now updated with the manuscript.
I also cleaned the data directory and a drafted readme tp address issue #63. 
I would recommend a separate PR for completing the draft I started here.

The test is failing because of the Brainbox URL security. This is likely an external issue so we might want to manually deploy the book and ignore this failed test. (see #64)

**Review by Remi-Gau — APPROVED** [review]
Cool.

LGTM.

Will use that to generate the final list of authors and update (manually) all the stuff downstream.

**Comment by Remi-Gau** [comment]
Everyone stand back! we are MERGING !!!!

# :rocket:

**Pull request merged**